docs/source/getting-started/institutional-faq.md

```diff
@@ -0,0 +1,265 @@
+* **JupyterHub** is an application that manages interactive computing sessions for **multiple users**.
+  It also connects them with infrastructure those users wish to access. It can provide
+  remote access to Jupyter Notebooks and Jupyter Lab for many people.
+
+# For management
+
+## Briefly, what problem does JupyterHub solve for us?
+
+JupyterHub provides a shared platform for data science and collaboration.
+It allows users to utilize familiar data science workflows (such as the scientific python stack,
+the R tidyverse, and Jupyter Notebooks) on institutional infrastructure. It also allows administrators
+some control over access to resources, security, environments, and authentication.
+
+## Is JupyterHub mature? Why should we trust it?
+
+Yes - the core JupyterHub application recently
+reached 1.0 status, and is considered stable and performant for most institutions.
+JupyterHub has also been deployed (along with other tools) to work on
+scalable infrastructure, large datasets, and high-performance computing.
+
+## Who else uses JupyterHub?
+
+JupyterHub is used at a variety of institutions in academia,
+industry, and governmental research labs. It is most-commonly used by two kinds of groups:
+
+* Small teams (e.g., data science teams, research labs, or collaborative projects) to provide a
+  shared resource for interactive computing, collaboration, and analytics.
+* Large teams (e.g., a department, a large class, or a large group of remote users) to provide
+  access to organizational hardware, data, and analytics environments at scale.
+
+Here are a sample of organizations that use JupyterHub:
+
+* **Universities and colleges**: UC Berkeley, UC San Diego, Cal Poly SLO, Harvard University, University of Chicago,
+  University of Oslo, University of Sheffield, Université Paris Sud, University of Versailles
+* **Research laboratories**: NASA, NCAR, NOAA, the Large Synoptic Survey Telescope, Brookhaven National Lab,
+  Minnesota Supercomputing Institute, ALCF, CERN, Lawrence Livermore National Laboratory
+* **Online communities**: Pangeo, Quantopian, mybinder.org, MathHub, Open Humans
+* **Computing infrastructure providers**: NERSC, San Diego Supercomputing Center, Compute Canada
+* **Companies**: Capital One, SANDVIK code, Globus
+
+See the [Gallery of JupyterHub deployments](../gallery-jhub-deployments.md) for
+a more complete list of JupyterHub deployments at institutions.
+
+## How does JupyterHub compare with hosted products, like Google Colaboratory, RStudio.cloud, or Anaconda Enterprise?
+
+JupyterHub puts you in control of your data, infrastructure, and coding environment.
+In addition, it is vendor neutral, which reduces lock-in to a particular vendor or service.
+JupyterHub provides access to interactive computing environments in the cloud (similar to each of these services).
+Compared with the tools above, it is more flexible, more customizable, free, and
+gives administrators more control over their setup and hardware.
```

Comment:
nit: is "more flexible, more customizable" the same as "more control over the setup"? Or do we want to make sure this point is made so we repeat it because repeating our point is a good way to make sure it is understood?

Reply:
In my mind, "customizable" is something you can do to the deployment after it's been set up initially, whereas "more flexible" means you can set it up initially on many different kinds of hardware etc. Perhaps that's not coming across?